src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -199,6 +204,20 @@ public virtual Publisher.IPublisherClient GrpcClient
             get { throw new NotImplementedException(); }
         }
 
+        /// <summary>
+        /// Creates the given topic with the given name.
+        /// </summary>
+        /// <param name="name">
+        /// The name of the topic. It must have the format
+        /// `"projects/{project}/topics/{topic}"`. `{topic}` must start with a letter,
+        /// and contain only letters (`[A-Za-z]`), numbers (`[0-9]`), dashes (`-`),
+        /// underscores (`_`), periods (`.`), tildes (`~`), plus (`+`) or percent
+        /// signs (`%`). It must be between 3 and 255 characters in length, and it
+        /// must not start with `"goog"`.
```

Comment:
That shouldn't be in this PR - it's documentation generated from pubsub.proto. Will try to work out why there are two commits in this PR instead of just one...